plugins/auth/index.js

```diff
@@ -83,13 +85,20 @@ exports.register = (server, options, next) => {
      * Generates a jwt that is signed and has a 2h lifespan
      * @method generateToken
      * @param  {Object} profile Object from generateProfile
+     * @param  {Integer} expires_sec JWT Expires time (must be seconds)
      * @return {String}         Signed jwt that includes that profile
      */
-    server.expose('generateToken', profile => jwt.sign(profile, pluginOptions.jwtPrivateKey, {
-        algorithm: ALGORITHM,
-        expiresIn: EXPIRES_IN,
-        jwtid: uuid()
-    }));
+    server.expose('generateToken', (profile, expiresSec = EXPIRES_IN) => {
+        const expiresIn = (expiresSec <= MAX_EXPIRES_IN)
+            ? expiresSec + EXPIRES_BUFFER
+            : MAX_EXPIRES_IN;
+
+        return jwt.sign(profile, pluginOptions.jwtPrivateKey, {
+            algorithm: ALGORITHM,
+            expiresIn,
```

Comment:
`expiresIn` arguments of `jwt.sign() ` can be set by seconds.
https://www.npmjs.com/package/jsonwebtoken

Reply:
Missing label? `expiresIn: expiresIn,`

Reply:
When label and value is same, it can be omitted in ES6.